src/menusys.c

```diff
@@ -925,14 +925,21 @@ void menuHandleInputMenu()
     }
 }
 
-void menuRenderMain()
+static void menuRenderDevicePage(theme_element_t *elem)
```

Comment:
I think a better name would be something like `menuRenderElements`.

Reply:
Sure I will change it